### PR TITLE
Add example for running the NeMo Framework

### PR DESCRIPTION
**Pull request by samskillman**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: examples/machine-learning/nemo-framework/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+README
+======
+
+1. Set up NeMo Framework Container
+
+This makes a few environment variable modifications to the
+nemo:23.11.framework container, and submits a slurm job to copy the framework
+launcher scripts and a few other auxilery files into your working directory. 
+
+	sbatch setup_nemo.sh
```
Comment (tpdownes) [inline examples/machine-learning/nemo-framework/README.md:10]: The behavior of large indents differs for single line and multiline strings. I suggest just using triple-backtick code blocks.

**Review by tpdownes — CHANGES_REQUESTED** [review]

File: examples/machine-learning/nemo-framework/README.md (hunk 1 above)
Comment (samskillman) [reply examples/machine-learning/nemo-framework/README.md]: Fixed

File: examples/machine-learning/nemo-framework/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+README
+======
+
+1. Set up NeMo Framework Container
+
+   This makes a few environment variable modifications to the [nvcr.io/nvidia/nemo:23.11.framework](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo)
+   container, and submits a Slurm job to copy the framework launcher scripts and a
+   few other auxilery files into your working directory.
```
Comment (cboneti) [inline examples/machine-learning/nemo-framework/README.md:8]: typo: auxiliary

Comment (samskillman) [reply examples/machine-learning/nemo-framework/README.md]: fixed.

**Review by tpdownes — APPROVED** [review]

**Pull request merged**